music21/midi/translate.py

```diff
@@ -522,25 +583,35 @@ def midiEventsToChord(eventList, ticksPerQuarter=None, inputM21=None):
         <music21.midi.MidiEvent NOTE_ON, track=1, channel=None, pitch=46, velocity=94>]
 
     Changed in v.7 -- Uses the last DeltaTime in the list to get the end time.
+    Changed in v.7.3 -- Returns a :class:`~music21.percussion.PercussionChord` if
+    any event is on channel 10. Returns None if `inputM21` provided.
+
+    >>> me2.channel = 10
+    >>> midi.translate.midiEventsToChord([dt1, me1, dt2, me2, dt3, me3, dt4, me4])
+    <music21.percussion.PercussionChord unpitched[A2] unpitched[B2]>
     '''
     tOn: int = 0  # ticks
     tOff: int = 0  # ticks
 
     if ticksPerQuarter is None:
         ticksPerQuarter = defaults.ticksPerQuarter
 
-    from music21 import pitch
     from music21 import volume
-    pitches = []
+    pitches: List[pitch.Pitch] = []
     volumes = []
 
+    firstOn: Optional['music21.midi.MidiEvent'] = None
+    any_channel_10 = False
     # this is a format provided by the Stream conversion of
     # midi events; it pre groups events for a chord together in nested pairs
     # of abs start time and the event object
     if isinstance(eventList, list) and eventList and isinstance(eventList[0], tuple):
         # pairs of pairs
         for onPair, offPair in eventList:
             tOn, eOn = onPair
+            if firstOn is None:
+                firstOn = eOn
+            any_channel_10 |= eOn.channel == 10
```

Comment:
Please rewrite to not use bitwise in-place with a comparison:

```
if not any_channel_10 and eOn.channel == 10:
     any_channel_10 = True
```
or even just 
```
if eOn.channel == 10: 
    any_channel_10 = True
```

setting a variable multiple times is faster than trying to figure out what this code does.